translations/code-of-conduct.portuguese.md

```diff
@@ -0,0 +1,71 @@
+* Ser respeitoso frente à pontos de vista e experiências diversas
+* Aceitar, gentilmente, críticas construtivas
+* Focar no que é melhor para a comunidade
+* Mostrar empatia aos outros membros da comunidade
+
+Exemplos de comportamentos inaceitáveis, incluem:
+
+* Uso de linguagem ou imagens de natureza sexual e investidas sexuais não bem vindas
+* Zueira, e comentários insultosos, e ataques pessoais e/ou políticos
+* Assédio público e/ou privado
+* Publicar informações privadas de outrem, sejam endereços físicos ou eletrônicos, sem expressa autorização
+* Outras conditas que possam ser razoavelmente consideradas inapropriadas em um ambiente profissional
+
+## Nossas responsabilidades
+
+Os mantenedores de projetos são responsáveis por deixar claros os padrões de aceitabilidade de comportamento,
+espera-se deles que ações justas e apropriadas sejam tomadas em resposta e qualquer tipo de comportamento inadequado.
+
+Mantenedores e projetos possuem o direito de remover, editar ou recusar comentários _commits_, código, edições na wiki
+_issues_, e outras contribuições não alinhadas com o Código de Conduta, ou banir permanente ou temporariamente qualquer
+contribuidor(a) por outros comportamentos que julgarem inapropriados, ameaçadores, ofensivos ou danosos.
+
+## Escopo
+
+Esse Cóigo de Conduta se aplica tanto em espaço de projetos quanto em espaços públicos
+quando uma pessoa estiver representando o projeto ou sua comundiade. Exemplos de representação de um projeto ou
+da comunidade incluem a utilização de um e-mail oficial do projeto, postagem via conta de rede social ou agir como
+um representante nomeado seja em eventos online ou offline. A representação de um projeto deve ser definida e esclarecida
+pelos mantenedores do projeto.
+
+## Execução
+
+Instâncias de comportamentos abusivos, agressivos ou inaceitáveis de outras formas deverão ser reportados
+através do e-mail do time de projeto community@hasura.io. Todos os comportamentos serão revisados e investigados e irão
+resultar em uma resposta que se julgar necessária e apropriada para as circunstâncias
+O time de projeto é obrigado a manter confidenciabilidade em relação a quem tive reportado o incidente.
+Detalhes posteriores acerca de específicos endurecimentos na política serão postados separadamente.
+
+Mantenedores de projeto que não seguirem ou apoiarem o Código de Conduta de boa fé irão enfrentar repercussões temporárias ou
+permanentementes de acordo com a determinação de outros membros da liderança de projeto.
+
+## Atribuições
+
+Esse Código de Conduta é uma adaptação do [Contributor Covenant][homepage], na versão 1.4,
+disponível em https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+[homepage]: https://www.contributor-covenant.org
+
+## Traduções
+
+Esse documento está disponível nas seguintes traduções:
+
+- [French :fr:](translations/code-of-conduct.french.md)
+- [Portuguese :pt:](translations/code-of-conduct.portuguese.md)
```

Comment:
Could you please remove the link to the Portuguese version (as this is the document you're referring to)? Also, could you add the link to the Portuguese translation to the English version of `cope-of-conduct.md`?